.github/workflows/replay-verify-mainnet.yaml

```diff
@@ -6,27 +6,27 @@ on:
       GIT_SHA:
         required: false
         type: string
-        description: The git SHA1 to test. If not specified, Forge will check the latest commits on the current branch
+        description: The git SHA1 to test. If not specified, it will use the latest commit on main.
   schedule:
     - cron: "0 9/12 * * *"
   pull_request:
     paths:
-      - ".github/workflows/nightly-replay-verify.yaml"
-
-env:
-  BUCKET: aptos-mainnet-backup-backup-831a69a8
-  SUB_DIR: e1
-  HISTORY_START: 0
-  TXNS_TO_SKIP: 12253479 12277499
-  CONFIG_TEMPLATE_NAME: s3-public.yaml
+      - ".github/workflows/replay-verify-mainnet.yaml"
 
 jobs:
   replay-transactions:
-    timeout-minutes: 720
+    # if we're running on a PR, it's only for testing purposes, so we can set a shorter timeout
+    timeout-minutes: ${{ github.event_name == 'pull_request' && 20 || 720 }}
     runs-on: high-perf-docker-with-local-ssd
     steps:
       - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3
         with:
           ref: ${{ inputs.GIT_SHA }}
-          fetch-depth: 0 # get all the history because cargo xtest --change-since origin/main requires it.
+
       - uses: ./.github/actions/replay-verify
+        with:
+          BUCKET: aptos-mainnet-backup-backup-831a69a8
+          SUB_DIR: e1
+          HISTORY_START: 0
```

Comment:
Is it possible to make this a variable where we accept an input from the UI? but default to 0?

Reply:
Yea it's possible. How often do we expect this to change tho?

Reply:
When debugging you might want to be able to change that a lot, Alden and Runtian were doing this by making PR before because this was hardcoded in the workflow so i think its worth exposing if we can have default as well

Reply:
Gotcha ok. I've made it configurable now via more `workflow_dispatch.inputs`. In a future PR, I think I might consider making replay-verify (and module-verify) reusable workflows, rather than composite actions. That way, we can manually trigger the base workflow with all the customizations, rather than editing or calling the `testnet` or `mainnet` variants of the workflow.